docs: Updated and expanded alert documentation (#137)
paradedb/charts: charts/paradedb/docs/runbooks/CNPGClusterZoneSpreadWarning.md

-------

The `CNPGClusterZoneSpreadWarning` alert is raised when pods are not evenly distributed across availability zones. To be
more accurate, the alert is raised when both of the following conditions are met:

> * the number of pods exceeds the number of zones

Reviewer: Will this flare up when a customer has 4 pods and 3 zones? We have a customer that wants to do this soon.

kubectl get nodes --label-columns topology.kubernetes.io/zone
```

> You can identify the current primary instance or use the following command:

Reviewer: ```suggestion
You can identify the current primary instance with the following command:
```
